test/cli/run/run-crash-handler.test.ts

```diff
@@ -33,6 +35,11 @@ test("raise ignoring panic handler does not trigger the panic handler", async ()
     ]),
   });
 
+  await proc.exited;
+
+  /// Wait two seconds for a slow http request, or continue immediatly once the request is heard.
+  await Promise.race([resolve_handler.promise, Bun.sleep(2000)]);
```

Comment:
2 seconds is not sufficient on IO busy CI machines. Can we just remove the manual timeout and let the test itself timeout?

Reply:
please re-read this test, it is waiting 2 seconds of threashold, and **passing** after that timer. the other promise is used to end the test (and fail) as soon as it hears a response.

if this test is still failing then there is actually broken underlying behavior i cannot reproduce locally.